pkgs/development/libraries/numcpp/default.nix

```diff
@@ -0,0 +1,46 @@
+{
+  lib,
+  stdenv,
+  fetchFromGitHub,
+  cmake,
+  boost181,
+  python3,
+  gtest,
+  eigen,
+  opencv,
+}:
+let
+  boost = boost181.override { enableStatic = true; };
+in
+stdenv.mkDerivation (finalAttrs: {
+  pname = "numcpp";
+  version = "2.11.0";
+
+  src = fetchFromGitHub {
+    owner = "dpilger26";
+    repo = "NumCpp";
+    rev = "Version_${finalAttrs.version}";
+    hash = "sha256-IAku1bcaMkawZxpQbvxcS6VX07ogw4UGo1DX2Wa8xwU=";
+  };
+
+  nativeBuildInputs = [cmake gtest python3 eigen opencv];
```

Comment:
Inputs that are only needed for the tests should be added to `nativeCheckInputs`.

Reply:
okay I moved to that